src/vmm/src/vstate/system.rs

```diff
@@ -50,7 +50,7 @@ impl KvmContext {
         let kvm = Kvm::new()?;
 
         // Check that KVM has the correct version.
-        if kvm.get_api_version() != KVM_API_VERSION as i32 {
+        if kvm.get_api_version() != i32::try_from(KVM_API_VERSION).unwrap() {
             return Err(Error::ApiVersion(kvm.get_api_version()));
         }
```

Comment:
Here we can allow casting, because `get_api_version` supposed to return same value(12) and we can just cast it to u32.

Reply:
I've updated that to add an allow.